packages/dnb-eufemia/src/elements/typography/P.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * HTML Element
+ *
+ */
+
+import React from 'react'
+import classnames from 'classnames'
+import { SpacingProps } from '../../components/space/types'
+import { DynamicElement } from '../../shared/types'
+import E from '../Element'
+
+export type PSize =
+  | 'x-small'
+  | 'small'
+  | 'basis'
+  | 'medium'
+  | 'large'
+  | 'x-large'
+  | 'xx-large'
+
+export type PProps = SpacingProps &
+  React.HTMLAttributes<HTMLParagraphElement> & {
+    element?: DynamicElement & 'p'
+    small?: boolean
```

Comment:
I think we should add docs (comments) each prop.

```suggestion
    /**
     * Formatted like other TS components (e.g. breadcrumb)
     */
    small?: boolean
```

Reply:
Sounds good, but I'm not sure if I understand the props well enough to properly document them 😅

Reply:
Lets do it in follow up PR. Try first by yourself with an PR. We will then double check for sure. You can also check existing documentation. Like for the level, we have the other Heading component, witch also supports `level`.